### ISSUE TITLE
Windows 11 - Can´t open Putty, user configs don't change like stated in the documentation

### ISSUE THREAD

**Issue by Luis-txd** [iterate-ch/cyberduck#15665]
I am trying to open PuTTY when connected to a server, and the "Open in PuTTy" menu option always opens the normal terminal and crashes immediately.

I habe tried changing the config file as such:

`<setting name="terminal.openssh.enable" value="false" />
<setting name="terminal.windowssubsystemlinux.enable" value="false" />
<setting name="terminal.command.ssh" value="C:\Program Files\PuTTY\putty.exe" /> `

However, the configuration always resets when closing and opening the application. 
I am using a Windows 11 device and the latest version of Cyberduck. Any help would be appreciated.

**Comment by AliveDevil** [comment]
Did you mean to close the ticket immediately?

Regardless, I can see that starting the ssh-client from Cyberduck using the Microsoft Store option immediately hangs up.

- Do you use the Microsoft Store Cyberduck app?
- Settings are always overwritten on application exit.
Make sure Cyberduck is closed before editing the user config.

**Comment by Luis-txd** [comment]
Yes i meant to close it. After i completed the processed like you specified in bullet point 2, it worked correctly. I appreciate the help
